.github/workflows/main.yml

```diff
@@ -399,7 +399,7 @@ jobs:
           echo "OBS_GIT_TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
 
       - name: Build Flatpak Manifest
-        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v4
+        uses: flatpak/flatpak-github-actions/flatpak-builder@master
```

Comment:
Switching to master is a bad idea, we should stick with v4.

Reply:
And yes, for stability version tags are better